internal/alerts/ci-prod-aks-mac-weu.json

```diff
@@ -1,277 +1,413 @@
-//                         "for": "PT3M",
-//                         "labels": {
-//                             "cluster": "/subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourcegroups/ci-prod-aks-mac-weu-rg/providers/microsoft.containerservice/managedclusters/ci-prod-aks-mac-weu"
-//                         },
-//                         "annotations": {
-//                             "description": "Memory usage % greater than 90 for prometheus-collector containers on cluster /subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourcegroups/ci-prod-aks-mac-weu-rg/providers/microsoft.containerservice/managedclusters/ci-prod-aks-mac-weu"
-//                         },
-//                         "severity": 4,
-//                         "resolveConfiguration": {
-//                             "autoResolved": true,
-//                             "timeToResolve": "PT10M"
-//                         },
-//                         "actions": [
-//                             {
-//                                 "ActionProperties": {
-//                                     "Icm.Enabled": "True"
-//                                 }
-//                             }
-//                         ]
-//                     }
-//                 ]
-//             }
-//         }
-//     ]
-// }
\ No newline at end of file
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {},
+    "variables": {},
+    "resources": [
+        {
+            "name": "containerinsights_mac_alerts",
+            "type": "Microsoft.AlertsManagement/prometheusRuleGroups",
+            "apiVersion": "2021-07-22-preview",
+            "location": "westeurope",
+            "properties": {
+                "description": "rule group for cluster /subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourcegroups/ci-prod-aks-mac-weu-rg/providers/microsoft.containerservice/managedclusters/ci-prod-aks-mac-weu in MAC: /subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourceGroups/ci-prod-aks-mac-weu-rg/providers/Microsoft.Monitor/accounts/ci-prod-aks-weu-mac",
+                "scopes": [
+                    "/subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourceGroups/ci-prod-aks-mac-weu-rg/providers/Microsoft.Monitor/accounts/ci-prod-aks-weu-mac"
+                ],
+                "rules": [
+                    {
+                        "alert": "up metric missing for target = node in cluster /subscriptions/9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb/resourcegroups/ci-prod-aks-mac-weu-rg/providers/microsoft.containerservice/managedclusters/ci-prod-aks-mac-weu",
```

Comment:
Hi Soham, would it be useful to also have these up metrics in the dashboards?

Reply:
Yes, this is a good idea. Let me do that update.